src/views/account/CreateSchool.vue

```diff
@@ -46,6 +46,12 @@
         methods: {
             createSchool: function () {
                 this.clickAllowed = false
+                api.school.isNameAvailable(this.createSchoolForm.schoolName).then(nameAvailable => {
```

Comment:
Will this block the form submission in case if validation failed (school name is not available)? Or it will just show a pop-up and the form submission will still be available?